dwave/preprocessing/presolve/cypresolve.pyx

```diff
@@ -60,12 +78,20 @@ cdef class cyPresolver:
 
     def apply(self):
         """Apply any loaded presolve techniques to the held constrained quadratic model."""
-        self.cpppresolver.apply()
+        try:
+            self.cpppresolver.normalize()
+        except RuntimeError as err:
+            # The C++ InvalidModelError is interpreted by Cython as a RuntimeError
+            # We could put in a bunch of code to reinterpret it, but because this
+            # is the only error type that should be raised by normalize() we just
+            # do it naively. 
+            raise InvalidModelError(err)
```

Comment:
What will `.feasibility()` return at this point? Does it matter?

Reply:
It will still be `Unknown`. We could set it to `Infeasible` but IMO in the error case it doesn't matter.